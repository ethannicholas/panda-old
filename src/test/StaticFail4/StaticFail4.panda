class StaticFail4 {
    @static
    method noOverriding() {
        Console.writeLine("You can't change me!")
    }
}

class Test : StaticFail4 {
    @override
    method noOverriding() {
        Console.writeLine("We'll see about that!")
    }
}