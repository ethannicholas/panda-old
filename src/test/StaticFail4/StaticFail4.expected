error: StaticFail4.panda:10:5: corresponding method in 'StaticFail4' is static and may not be overridden
1 error
